tests/displays/cavity_display/frontend/test_fault_count_display.py

```diff
@@ -18,6 +18,15 @@ def window():
     yield FaultCountDisplay(lazy_fault_pvs=True)
 
 
```

Comment:
@hmarts9 or @lisazacarias . I have attempted to make a test! Did I actually test what's supposed to be tested or over test or miss stuff?

Reply:
I think what I would've done was more check to make sure that if you suppress a fault, it no longer shows up in the data to be plotted

Reply:
Okay, the original `make_tlc_list` was just a little new function I made to declutter my init, and since it's a new function, I figured the rule was it gets a test.
You're right though, I still need to add the checks for suppressing a fault in `test_get_data_without_pot`

Reply:
Does the `test_get_data_with_tlc_omitted` function later in the code address the test for checking that data no longer shows up for a suppressed fault? I didn't change much in that function, so I suspect there is more to be done